server/src/main/java/io/druid/query/extraction/LookupExtractionModule.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.extraction;
+
+import com.fasterxml.jackson.annotation.JacksonInject;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.net.HostAndPort;
+import com.google.inject.Binder;
+import com.google.inject.Inject;
+import com.metamx.common.ISE;
+import com.metamx.common.logger.Logger;
+import io.druid.curator.announcement.Announcer;
+import io.druid.guice.Jerseys;
+import io.druid.guice.JsonConfigProvider;
+import io.druid.guice.LifecycleModule;
+import io.druid.guice.annotations.Json;
+import io.druid.guice.annotations.Self;
+import io.druid.guice.annotations.Smile;
+import io.druid.initialization.DruidModule;
+import io.druid.server.DruidNode;
+import io.druid.server.initialization.ZkPathsConfig;
+import io.druid.server.listener.announcer.ListenerResourceAnnouncer;
+import io.druid.server.listener.announcer.ListeningAnnouncerConfig;
+import io.druid.server.listener.resource.AbstractListenerHandler;
+import io.druid.server.listener.resource.ListenerResource;
+import io.druid.server.namespace.cache.LookupCoordinatorManager;
+import org.apache.curator.utils.ZKPaths;
+
+import javax.ws.rs.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Include when a service can listen to lookups
+ */
+public class LookupExtractionModule implements DruidModule
+{
+  private static final String PROPERTY_BASE = "druid.zk.paths";
```

Comment:
this is the normal path used in other configs. Moved to global static string to highlight this.